web/profiles/custom/yalesites_profile/modules/custom/ys_themes/ys_themes.libraries.yml

```diff
@@ -2,3 +2,9 @@ ys_themes:
   css:
     theme:
       css/ys-themes.css : {}
+levers:
+  css:
+    theme:
+      css/levers.css : {}
+  js:
+    js/levers.js: {}
```

Comment:
I think you need to add a dependency to 'Drupal' since you are using it in the JS. Something else (likely the toolbar) is already loading it, so there is not failure at the moment. But we should still add it here.
```
dependencies:
    - core/drupal
```

Reply:
Added dependencies.